src/style.css

```diff
@@ -0,0 +1,137 @@
+body {
+  margin: 0 ;
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  width: 200px;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+p,
+h2 {
+  margin: 0;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.card__text {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: #616070;
+  font-style: normal;
+  margin-bottom: 16px;
+
+}
+
+/* #region Stars */
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+
+}
+
+.stars {
+  display: inline-flex;
+}
+
+.stars__star {
+  align-items: center;
+  width: 16px;
+  height: 16px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+
+}
+
+.stars--1 > .stars__star:nth-child(1),
+.stars--2 > .stars__star:nth-child(-n+2),
+.stars--3 > .stars__star:nth-child(-n+3),
+.stars--4 > .stars__star:nth-child(-n+4),
+.stars--5 > .stars__star:nth-child(-n+5) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--4 .stars__star:not(:last-child) {
+  margin-right: 4px;
+}
```

Comment:
Split your css code to different files - BEM blocks.

According to [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)

`11.[FILE STRUCTURE] - You should create separate file for each BEM block`

detailed info how to do it in [this video](https://mate.academy/learn/css-advanced/bem#/video/917)